### PR TITLE
corrected styling for Subjects and Claims selections

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
Also, now all subject selectors now have a vertical display (like Claims did).

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/claim-report-form.component.html [hunk 1]
```diff
@@ -145,16 +145,15 @@ <h3>{{'aggregate-report-form.section.assessment-heading' | translate}}</h3>
...
-                                 name="subjects"
-                                 [vertical]="true"
-                                 analyticsEvent="FilterClick"
-                                 analyticsCategory="AggregateQueryBuilder"
-                                 [allOptionAnalyticsProperties]="{label: 'Subject: All'}"
-                                 [options]="filteredOptions.subjects"
-                                 [(ngModel)]="settings.subjects"
-                                 [ngModelOptions]="{standalone: true}"
-                                 (change)="onSubjectsChange()"></sb-button-group>
+                <sb-checkbox-group id="subjects"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/claim-report-form.component.html]: is this to fix the styling? we don't know why button group styles don't work and checkbox group styles do?

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/aggregate-report/query-forms/claim-report-form.component.html]: hmm, I might have just needed to put type="checkbox" on this. I forgot sb-checkbox-group was deprecated

File: webapp/src/main/webapp/src/styles.less [hunk 2]
```diff
@@ -429,6 +429,20 @@ body .select-menu-group {
   }
 }
 
+.nested-btn-group {
+  > a:first-child {
+    display: table-cell;
+  }
+  &.vertical {
+    > a:first-child {
```
[the remark this answers is not in the document]
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/styles.less]: Yes, and I had to add border-top/border-bottom radiuses. Not sure where those are applied, but they are missing from anchors

[the remark this answers is not in the document]
Comment (esotrope) [reply webapp/src/main/webapp/src/styles.less]: ok cool. can you also add a comment that this should be moved to ui kit so we can sort it out later

File: webapp/src/main/webapp/src/app/aggregate-report/query-forms/target-report-form.component.html [hunk 3]
```diff
@@ -68,14 +68,16 @@ <h3>{{'aggregate-report-form.section.assessment-heading' | translate}}</h3>
                 <label for="subject">{{'aggregate-report-form.field.subject-label' | translate}}</label>
               </div>
               <div class="col-md-6">
-                <sb-radio-group id="subject"
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/aggregate-report/query-forms/target-report-form.component.html]: I'll revert this change

File: webapp/src/main/webapp/src/app/shared/sb-toggle.component.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ const DefaultButtonGroupStyles = 'btn-group-sm';
 const DefaultButtonStyles = 'btn-primary';
 
 /**
- * @deprecated use <sb-button-group type="radio">
+ * @deprecated use <sb-radio-group>
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/shared/sb-toggle.component.ts]: `<sb-button-group type="radio">` no longer works

Comment (esotrope) [reply webapp/src/main/webapp/src/app/shared/sb-toggle.component.ts]: : / that's not good

**Pull request merged**